gui/next/src/lib/ui/ConnectionIndicator.svelte

```diff
@@ -22,62 +20,24 @@ let onlineCheckInterval;
 onMount(async () => {
   checkIfOnline();
 
-  clearInterval(onlineCheckInterval);
   onlineCheckInterval = setInterval(checkIfOnline, checkIntervalTime);
+  return () => clearInterval(onlineCheckInterval);
 });
 
-$: if($page.url.pathname.startsWith('/logs')){
-  clearInterval(onlineCheckInterval);
-
-  checkIntervalTime = 3000;
-  onlineCheckInterval = setInterval(checkIfOnline, checkIntervalTime);
-} else {
-  if(checkIntervalTime === 3000){
-    clearInterval(onlineCheckInterval);
-
-    checkIntervalTime = 7000;
-    onlineCheckInterval = setInterval(checkIfOnline, checkIntervalTime);
-  }
-}
-
-// purpose:		checks if the app is connected to the instance by getting some api data
-// returns:		updates the $state.online store (bool) and returns a boolean response
+// purpose:		checks if the app is connected to the instance by fetching /info
+// effect:		updates the $state.online store (bool)
 // ------------------------------------------------------------------------
 const checkIfOnline = async () => {
   if(browser && document.visibilityState !== 'hidden'){
-    const last = $state.logs.logs?.at(-1)?.id ?? null;
-    const newLogs = await logs.get({ last: last });
-
-    // if already showing logs, just append the new ones
-    if(last){
-      if(newLogs.logs?.length){
-        $state.logs.logs = [...$state.logs.logs, ...newLogs.logs];
-
-        // store the times when logs were downloaded to mark new ones
-        $state.logs.downloaded_at.push(Date.now());
-        if($state.logs.downloaded_at.length > 2){
-          $state.logs.downloaded_at.splice(0, 1);
-        }
+    fetch('http://localhost:3333/info').then(response => response.json()).then(data => {
```

Comment:
@z-x Port should not be hardcoded.

Reply:
Yup, updated. I'll extract this in the near future not to worry about it each time.